eng/templates/stages/mirror.yml

```diff
@@ -112,3 +114,47 @@ stages:
         fi
       workingDirectory: $(Pipeline.Workspace)/$(RepoDir)
       displayName: Mirror and tag
+
+  - job: MirrorPackages
```

Comment:
I am not a fan of this being a separate job.  Agent allocation at it's worse can really be problematic and can really be a source of frustration during releases.